### PR TITLE
Uptime Code Fix, Helpful help, and some other Easter Eggs

### PR DESCRIPTION
**Pull request by tyraziel**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -136,31 +136,49 @@ client.on('messageCreate', async (message) => {
...
+                                  + '[[v:card_name]] - Will show the most verbose display for a '
+                                  + 'given card.');
+                break;
               case 'pint':
                 await message.reply('I\'ll raise a glass to that!  Not that bots can actually drink a pint....');
                 break;
-              case 'help':
-                await message.reply('I\'m sorry Dave, but I can\'t allow you to do that.');
+              case 'open':
+                await message.reply('I\'m sorry, Dave.  I\'m afriad I can\'t do that.');
```
Comment (ThijsMergaert) [inline index.js]: Typo

```suggestion
                await message.reply('I\'m sorry, Dave.  I\'m afraid I can\'t do that.');
```

File: index.js [hunk 2]
```diff
@@ -136,31 +136,49 @@ client.on('messageCreate', async (message) => {
...
+                break;
+              case 'cake':
+                await message.reply('The cake is not a lie.');
```
Comment (ThijsMergaert) [inline index.js:161]: But it is!

File: index.js (hunk 1 above)
Comment (tyraziel) [reply index.js]: Thanks!

File: index.js (hunk 2 above)
Comment (tyraziel) [reply index.js:161]: If you want to change it, that's fine :)

**Pull request merged**